CONTRIBUTING.md

```diff
@@ -11,6 +11,15 @@ expectations the Node Foundation requires of its contributors.
 * A **Contributor** is any individual creating or commenting on an issue or pull request.
 * A **Collaborator** is a subset of contributors who have been given write access to the repository.
```

Comment:
Subset makes it sound a bit too logical, and unnecessary. I would go with `A collaborator is a contributor who has been given write access to the repository.`